src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/model/JtsLayer.java

```diff
@@ -22,7 +22,7 @@
      * @throws IllegalArgumentException when {@code name} is null
      */
     public JtsLayer(String name) {
-        this(name, new ArrayList<>(0));
+        this(name, new ArrayList<Geometry>(0));
```

Comment:
Diamond operator should be supported?

Reply:
I'll put back.  If we are aiming for Android support rather than, say, JDK 6/7 then we are fine as long as we don't touch unsupported APIs.  Essentially Java 8 and avoiding unsupported APIs.